### PR TITLE
New recipe: stginga

### PR DESCRIPTION
**Pull request by pllim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
   - [x] @pllim (since this is PR author, extra comment about willingness is unnecessary?)
   - [x] @jhunkeler 
   - [x] @rendinam

### CODE REVIEW

**Comment by rendinam** [comment]
I am willing to be listed as a maintainer for this package.

**Comment by pllim** [comment]
@conda-forge/core -- Would be nice if the linter edits its original comment in-place instead of posting a new comment every time. I cannot hide the old ones, so it clutters up after a while. Please let me know if there is a proper repo for such a feature request. Thanks!

**Comment by beckermr** [comment]
We've discussed this feature before. See this issue: https://github.com/conda-forge/conda-forge-webservices/issues/276

TL;DR There seems to be a consensus to keep things as they are. Comments on that issue welcome!

**Comment by pllim** [comment]
@jhunkeler cannot say no given he is my coworker. This is good to merge. Thanks.

**Comment by beckermr** [comment]
hahahaha. He should still comment here. :)

**Comment by jhunkeler** [comment]
I am commenting here.
I consent.

:rocket:

**Comment by mwcraig** [comment]
Not a blocker, but if this can be `noarch: python` then you can add the Python version restriction with `python >=3.6` (or whatever minimum you want) in `host` and `run`.

**Comment by pllim** [comment]
@mwcraig , do I also have to remove `skip: true  # [py2k]` if I use `noarch`? I tried earlier but linter was complaining.

**Comment by pllim** [comment]
@mwcraig , I added noarch back. 🤞

**Comment by mwcraig** [comment]
Looks good, merging!

**Comment by pllim** [comment]
Thanks, everyone!
